docs/docs/operator-manual/plugins/examples.mdx

```diff
@@ -0,0 +1,111 @@
+# Examples
+
+This is a bundle of example usages of the built-in Plugins. Look at [custom plugins](/operator-manual/plugins/thirdparty)
+for how to build your own resources and customizing your setup even further.
+
+## Services as `NodePort`
+
+Some load-balancers needs to address Services as LoadBalancers, e.g. the AWS Load Balancer.
+
+The `rigdev.object_template` plugin is an easy way to change this behavior:
+
+```yaml
+config:
+  pipeline:
+    steps:
+      - plugins:
+          - name: rigdev.object_template
+            config: |
+              kind: Service  # Match services
+              object: |
+                spec:
+                  # Set the type to NodePort.
+                  type: NodePort
+        # Include this if needed for the Rig Platform as well.
+        enableForPlatform: true
+```
+
+## Conditional injection through Annotations
+
+Adding Annotations to a Capsule is an easy way to customize for which Capsules a given operation should be performed.
+
+In this example, we're going to inject an `terminationGracePeriodSeconds: 30` if, and only if,
+the annotation `example.com/long-grace-period` is set to `true` on the Capsule.
+
+```yaml
+config:
+  pipeline:
+    steps:
+      - plugins:
+          - name: rigdev.object_template
+            config: |
+              kind: Deployment
+              object: |
+                {{ with .capsule.metadata.annotations }}
+                {{ if eq (index . "example.com/long-grace-period") "true" }}
```

Comment:
It would be nice if we had a mechanism for doing general predicate stuff like this before handing off to the plugins. 

Eg. we could have in it in our pipeline API.